internal/cli/atlas/quickstart/confirm_cluster_setup.go

```diff
@@ -27,47 +27,27 @@ func (opts *Opts) askConfirmConfigQuestion() error {
 		return nil
 	}
 
-	diskSize := 0.5
-	if opts.newCluster().DiskSizeGB != nil {
-		diskSize = *opts.newCluster().DiskSizeGB
-	}
-
-	const noMsg = "No"
-	const yesMsg = "Yes"
-	loadSampleData := yesMsg
-	if opts.SkipSampleData {
-		loadSampleData = noMsg
-	}
-
-	runMongoShell := noMsg
-	if opts.runMongoShell {
-		runMongoShell = yesMsg
+	loadSampleData := ""
+	if !opts.SkipSampleData {
+		loadSampleData = `
+Load sample data:			Yes`
 	}
 
 	fmt.Printf(`
-[Summary of changes]
+[Confirm cluster settings]
 Cluster Name:				%s
-Cluster Tier:				%s
-Cloud Provider:				%s
-Region:					%s
-Cluster Disk Size (GiB):		%.1f
+Cloud Provider and region:		%s
```

Comment:
My bad, I just noticed here it should be Provider and Region (consistent with the GUI).

Reply:
Thanks updated.